profile/README.md

```diff
@@ -1,12 +1,43 @@
+<center>
+
+![UESTC Logo](https://www.uestc.edu.cn/c13209856342b0df2c7ee4eaeb07db5a.png)
```

Comment:
Can this image to be aligned into center?

Reply:
It seems the `center` element doesn't work, I'll try another way of making it center aligned.

Reply:
> Can this image to be aligned into center?

I set the whole title part center aligned.